media-source/mediasource-duration.html

```diff
@@ -44,7 +44,7 @@
                   test.waitForExpectedEvents(function()
                   {
                       assert_greater_than_equal(mediaElement.currentTime, seekTo, 'Playback time has reached seekTo');
-                      assert_false(mediaElement.seeking, 'mediaElement.seeking after seeked to seekTo');
+                      assert_false(mediaElement.seeking, 'mediaElement.seeking after sought to seekTo');
```

Comment:
The media element API uses seeked instead of sought intentionally because it's easier to spell, so all cases of seeked should remain.